app/controllers/home_controller.rb

```diff
@@ -1,4 +1,21 @@
 class HomeController < ApplicationController
   def index
+    if user_signed_in?
+      @post=Post.new
+      @posts=Post.all
+=begin
+      @posts.each do |post|
+        check=post.likes.where("user_id=#{current_user[:id]}")
+        if check
```

Comment:
remove the commented code. Please avoid this in future

Reply:
Done